app/styles/front.scss

```diff
@@ -688,21 +690,51 @@ $autre-lighter: #cce4e7;
   cursor: help;
 }
 
-[data-toggle="buttons"] {
-  display: flex;
-  flex-direction: row;
+legend {
+  border-bottom: 0px;
+}
 
-  button, label {
-    flex: 1;
-  }
+.radioButton {
+  @extend .btn-default;
+}
 
-  .active .help-block, .active .help-col {
-    color: #D8D8D8;
-  }
+.invisibleInput {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px; margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
+
+input:checked + .radioButton, input:checked + .radioButton:active:hover {
+  background-color: $btn-success-bg;
+  color: white;
+}
+
+input:checked + .radioButton > .help-col {
+  color: #D8D8D8;
+}
+
+.button-row {
+  display: flex;
+  flex-direction: row;
```

Comment:
IE 10 displays the blocks in a column but it is usable.

![image](https://cloud.githubusercontent.com/assets/11834997/16077242/f2842ccc-32f7-11e6-9d0e-166241f8baf5.png)